cypress/integration/seededFlows/adminFlows/users/manageUserOptions.spec.js

```diff
@@ -67,22 +67,22 @@ describe('Manage User Options', () => {
 
     it(`should merge a user's account with another account`, () => {
       openUserOptions(() => {
-        cy.findByRole('button', { name: 'Merge accounts' }).click();
+        cy.findByRole('button', { name: 'Merge users' }).click();
       });
 
       cy.getModal().within(() => {
         cy.findByRole('spinbutton', { name: 'User ID' }).type('3');
-        cy.findByRole('button', { name: 'Merge users' }).click();
+        cy.findByRole('button', { name: 'Merge and delete' }).click();
       });
     });
 
     it(`should banish a user for spam`, () => {
       openUserOptions(() => {
-        cy.findByRole('button', { name: 'Banish for spam' }).click();
+        cy.findByRole('button', { name: 'Banish user' }).click();
       });
 
       cy.getModal().within(() => {
-        cy.findByRole('button', { name: 'Banish User for spam' }).click();
+        cy.findByRole('button', { name: 'Banish Trusted User 1 :/' }).click();
```

Comment:
This is failing, as Cypress can't find this button. That said, I've tried a handful of things to try and get this to work, but for some reason it's not picking up this "banish" button's text. Any ideas on how to resolve this? 🤔

Reply:
```suggestion
        cy.findByRole('button', { name: 'Banish Trusted User 1 \:/' }).click();
```
I haven't tried this locally, but the output from the test failure lists this as the available button name ^ (just missing the backslash)

Reply:
```suggestion
        cy.findByRole('button', { name: 'Banish Trusted User 1 \\:/' }).click();
```

OK so what's happening is `\` is escaping the next character.. but `:` doesn't need escaped, so the linter removes it and we're left with `User 1 :/` when what we need is `User 1 \:/` 🙈 

If we escape the backslash like this it works as expected. Smh. What a weird one 😄  I've tested this locally and it all passed OK

Reply:
Oh my gosh! 🙈  Thank you sooooo much for the help, @aitchiss! 🙌